crates/router/src/connector/stripe/transformers.rs

```diff
@@ -64,19 +64,32 @@ pub enum Auth3ds {
 }
 
 #[derive(Debug, Eq, PartialEq, Serialize)]
-#[serde(rename_all = "snake_case")]
+#[serde(rename_all = "snake_case", untagged)]
```

Comment:
I guess you can use internally tagged enum type here https://serde.rs/enum-representations.html#internally-tagged